codespell_lib/data/dictionary.txt

```diff
@@ -24969,17 +25215,21 @@ vissible->visible
 visted->visited
 visting->visiting
 vistors->visitors
+visualizaion->visualization
+visualzation->visualization
 vitories->victories
 vitual->virtual
 viusally->visually
 viusualisation->visualisation
 viwer->viewer
 viwers->viewers
+vizualization->visualization, virtualization,
 vlarge->large
 vlaue->value
 vlaues->values
 vlues->values
 volatage->voltage
+volatge->voltage
```

Comment:
Would you mind adding a separate line for `volatges->voltages`?

Reply:
Added volatges->voltages